src/components/LibrariesListItem.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { Panel } from "react-bootstrap";
+import { LibraryData } from "../interfaces";
+import LibraryDetailContainer from "./LibraryDetailContainer";
+import { GenericWedgeIcon } from "@nypl/dgx-svg-icons";
+
+export interface LibrariesListItemProps {
+  library: LibraryData;
+  active: boolean;
+  select: (event: __React.MouseEvent) => void;
+}
+export interface LibrariesListItemState {
+  open: boolean;
+}
+
+export default class LibrariesListItem extends React.Component<LibrariesListItemProps, LibrariesListItemState> {
+  constructor(props: LibrariesListItemProps) {
+    super(props);
+    this.colorCode = this.colorCode.bind(this);
+  }
+
+  header() {
+    return (
+      <div onClick={this.props.select}>
+        <span>{this.props.library.name} ({this.props.library.short_name})</span>
+        <GenericWedgeIcon className={this.props.active ? "up-icon" : "down-icon"} />
+      </div>
+    );
+  }
+
+  colorCode() {
+    // If both stages are in production, background is green;
+    // if at least one stage is cancelled, background is red;
+    // otherwise, background is yellow.
+    let stages = [this.props.library.registry_stage, this.props.library.library_stage];
+
+    if (stages.every((stage) => stage === "production")) {
+      return "success";
+    }
+    else if (stages.some((stage) => stage === "cancelled")) {
```

Comment:
Is `cancelled` a new stage value for the library registry? I don't remember encountering that before for the circulation admin when working on it.

Reply:
I don't know how new it is (but it's definitely supposed to be there!)...